cfy_manager/utils/db.py

```diff
@@ -56,7 +56,7 @@ def get_psql_env_and_base_command(logger, db_key='cloudify_db_name',
         # or if we're installing a single database node,
         # "peer" authentication is used
         if pg_config['server_username'] == 'postgres':
-            base_command.extend(['-u', 'postgres'])
+            base_command.extend(['sudo', '-E', '-u', 'postgres'])
```

Comment:
this needs to be `/usr/bin/sudo`, not just `sudo`, because it is passed (in `cfy_manager dbs shell`) into an `os.execve`, and that doesn't go through PATH

Reply:
Good point, updating.